src/codeyCommand.ts

```diff
@@ -169,7 +170,7 @@ const setCommandOption = (
         )
       );
     default:
-      throw new Error(`Unknown option type.`);
+      throw new CodeyUserError(undefined, `Unknown option type.`);
```

Comment:
Actually, we should just change this to a regular Error, since this isn't an error propagated to the user.